recipes/biorbd-viz/yum_requirements.txt

```diff
@@ -0,0 +1,10 @@
+xorg-x11-server-Xvfb
+gtk2-devel
+libXScrnSaver
+libXtst
+alsa-lib
```

Comment:
Please check that there aren't conda forge packages for these already.  For example, I know we have an alsa package

Reply:
This list should be as small as possible

Reply:
Done :)
I could not send the 2 remaining into the meta.yaml